include/SFML/Audio/SoundStream.hpp

```diff
@@ -281,7 +281,8 @@ class SFML_AUDIO_API SoundStream : public SoundSource
 
     enum
     {
-        BufferCount = 3 ///< Number of audio buffers used by the streaming loop
+        BufferCount = 3,    ///< Number of audio buffers used by the streaming loop
+        BufferRetries = 2   ///< Number of retries (excluding initial try) for onGetData()
```

Comment:
Right! ^^"

So, do you (and others) feel this new constant should be moved or not?